zone/questmgr.cpp

```diff
@@ -4285,4 +4285,326 @@ std::string QuestManager::secondstotime(int duration) {
+		{ "Pale Violet Red1", "#F778A1" },
+		{ "Pale Violet Red2", "#E56E94" },
+		{ "Pale Violet Red3", "#C25A7C" },
+		{ "Pale Violet Red4", "#7E354D" },
+		{ "Pastel Green", "#00FF00" },
+		{ "Pink", "#FAAFBE" },
+		{ "Pink1", "#FF00FF" },
+		{ "Pink2", "#E7A1B0" },
+		{ "Pink3", "#C48793" },
+		{ "Pink4", "#7F525D" },
+		{ "Plum", "#B93B8F" },
+		{ "Plum1", "#F9B7FF" },
+		{ "Plum2", "#E6A9EC" },
+		{ "Plum3", "#C38EC7" },
+		{ "Plum4", "#7E587E" },
+		{ "Purple", "#8E35EF" },
+		{ "Purple1", "#893BFF" },
+		{ "Purple2", "#7F38EC" },
+		{ "Purple3", "#6C2DC7" },
+		{ "Purple4", "#461B7E" },
+		{ "Red", "#FF0000" },
+		{ "Red1", "#F62217" },
+		{ "Red2", "#E41B17" },
+		{ "Rosy Brown", "#B38481" },
+		{ "Rosy Brown1", "#FBBBB9" },
+		{ "Rosy Brown2", "#E8ADAA" },
+		{ "Rosy Brown3", "#C5908E" },
+		{ "Rosy Brown4", "#7F5A58" },
+		{ "Royal Blue", "#2B60DE" },
+		{ "Royal Blue1", "#306EFF" },
+		{ "Royal Blue2", "#2B65EC" },
+		{ "Royal Blue3", "#2554C7" },
+		{ "Royal Blue4", "#15317E" },
+		{ "Salmon1", "#F88158" },
+		{ "Salmon2", "#E67451" },
+		{ "Salmon3", "#C36241" },
+		{ "Salmon4", "#7E3817" },
+		{ "Sandy Brown", "#EE9A4D" },
+		{ "Sea Green", "#4E8975" },
+		{ "Sea Green1", "#6AFB92" },
+		{ "Sea Green2", "#64E986" },
+		{ "Sea Green3", "#54C571" },
+		{ "Sea Green4", "#387C44" },
+		{ "Sienna", "#8A4117" },
+		{ "Sienna1", "#F87431" },
+		{ "Sienna2", "#E66C2C" },
+		{ "Sienna3", "#C35817" },
+		{ "Sienna4", "#7E3517" },
+		{ "Sky Blue", "#82CAFF" },
+		{ "Sky Blue1", "#6698FF" },
+		{ "Sky Blue2", "#79BAEC" },
+		{ "Sky Blue3", "#659EC7" },
+		{ "Sky Blue4", "#41627E" },
+		{ "Slate Blue", "#357EC7" },
+		{ "Slate Blue1", "#737CA1" },
+		{ "Slate Blue2", "#6960EC" },
+		{ "Slate Blue3", "#342D7E" },
+		{ "Slate Gray", "#657383" },
+		{ "Slate Gray1", "#C2DFFF" },
+		{ "Slate Gray2", "#B4CFEC" },
+		{ "Slate Gray3", "#98AFC7" },
+		{ "Slate Gray4", "#616D7E" },
+		{ "Spring Green", "#4AA02C" },
+		{ "Spring Green1", "#5EFB6E" },
+		{ "Spring Green2", "#57E964" },
+		{ "Spring Green3", "#4CC552" },
+		{ "Spring Green4", "#347C2C" },
+		{ "Steel Blue", "#4863A0" },
+		{ "Steel Blue1", "#5CB3FF" },
+		{ "Steel Blue2", "#56A5EC" },
+		{ "Steel Blue3", "#488AC7" },
+		{ "Steel Blue4", "#2B547E" },
+		{ "Thistle", "#D2B9D3" },
+		{ "Thistle1", "#FCDFFF" },
+		{ "Thistle2", "#E9CFEC" },
+		{ "Thistle3", "#C6AEC7" },
+		{ "Thistle4", "#806D7E" },
+		{ "Turquoise", "#00FFFF" },
+		{ "Turquoise1", "#43C6DB" },
+		{ "Turquoise2", "#52F3FF" },
+		{ "Turquoise3", "#4EE2EC" },
+		{ "Turquoise4", "#43BFC7" },
+		{ "Violet", "#8D38C9" },
+		{ "Violet Red", "#F6358A" },
+		{ "Violet Red1", "#F6358A" },
+		{ "Violet Red2", "#E4317F" },
+		{ "Violet Red3", "#C12869" },
+		{ "Violet Red4", "#7D0541" },
+		{ "White", "#FFFFFF" },
+		{ "Yellow", "#FFFF00" },
+		{ "Yellow1", "#FFFC17" },
+		{ "Yellow Green", "#52D017" }
+	};
+	for (auto color : colors) {
+		if (!strcmp(str_tolower(color.first).c_str(), str_tolower(color_name).c_str())) {
```

Comment:
Using `strcasecmp` here instead would avoid having to create the two temporary strings with `str_tolower`